public/style/sass/_report.scss

```diff
@@ -1,3 +1,57 @@
+
```

Comment:
Seems to be some extra margin at the top of the form

GET RID OF IT!!!!

<img width="473" alt="screen shot 2017-05-10 at 12 09 56" src="https://cloud.githubusercontent.com/assets/15878887/25891415/a3f88294-3579-11e7-85dc-d2a9786d49d0.png">